test/ApplicationSpec.scala

```diff
@@ -0,0 +1,30 @@
+import org.specs2.mutable._
```

Comment:
Is this a test we wrote? I'm guessing no from the "Your new application is ready". Please remove.